evap/staff/importers.py

```diff
@@ -1,15 +1,17 @@
 from collections import OrderedDict, defaultdict
-import xlrd
 
 from django.conf import settings
 from django.db import transaction
+from django.db.models import Q
 from django.utils.translation import ugettext_lazy, ugettext as _
 from django.utils.safestring import mark_safe
 from django.core.exceptions import ValidationError
 
 from evap.evaluation.models import Contribution, Course, CourseType, Degree, Evaluation, UserProfile
 from evap.evaluation.tools import clean_email
 
+import xlrd
```

Comment:
this should actually stay on top (see https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#imports)

Reply:
Ah, my fault then - codacy complained about it (just as it does with other files, check this out: https://app.codacy.com/manual/evap/EvaP/file/41013863394/issues/source?bid=7257767&fileBranchId=7257767)
Can we fix this? From the message, it seems to expect the same order as django does, but it probably can't detect xlrd as being third-party. Codacy does not seem to have an option to install the requirements before running pylint. If we can't fix it, I'd suggest turning off the import order linting rule. I observe myself ignoring codacy more and more due to false negatives.

Reply:
i'm fine with turning it off. we could instead use the same approach as 1327 to enforce correct linting (running https://github.com/fsr-de/1327/blob/master/_1327/main/management/commands/lint.py in https://github.com/fsr-de/1327/blob/master/.travis.yml)

Reply:
Added a comment on #1092

Problem is that the rule itself as it is configured in the .pylintrc is probably a good thing (running a linter locally should tell you this), it's just that codacy doesn't install the requirements thus pylint can not detect xlrd as a third party module.